src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -127,6 +127,19 @@ class IOUAmountPage extends React.Component {
         return amount.replace(/,/g, '');
     }
 
+    /**
+     * Adds a leading zero to amount if user entered just the decimal separator
+     *
+     * @param {String} amount - Changed amount from user input
+     * @returns {String}
+     */
+    addLeadingZero(amount) {
+        if (amount === '.') {
+            return '0.';
+        }
+        return amount;
```

Comment:
We can clean up the code here.
```suggestion
        return amount === '.' ? '0.' : amount;
```

Reply:
> We can clean up the code here.

Thanks, updated